packages/app/components/Today/OpenQueueCard.tsx

```diff
@@ -274,11 +278,24 @@ export default OpenQueueCard;
 
 export function OpenQueueCardSkeleton(): ReactElement {
   return (
-    <PaddedCard>
-      <Skeleton paragraph={{ rows: 2 }} />
-      <Row justify="space-between" align="bottom">
-        <Skeleton.Avatar size={96} />
+    <PaddedCard
+      headStyle={{
+        background: "#25426C",
+        color: "#FFFFFF",
+        borderRadius: "6px 6px 0 0",
+      }}
+      className={"open-queue-card"}
```

Comment:
if this file now considers closed queues, should we rename the file from `OpenQueueCard` to just `QueueCard` for slightly more clarity?